english/dream/dream.md

```diff
@@ -1,6 +1,6 @@
-You understand that it's difficult to wake up from a dream if you think this is all really happening to you.
+You understand that it's difficult to wake up from a dream if you think this is all really happening to you. See, this is the thing with dreams. If you don't pay attention to strange things and realize it's a dream, you cannot control it. And if you pay too much attention to the fact that it's a dream, you wake up. The trick is to find that sweet spot between being asleep yet awake enough to notice it is a dream. Like balancing on a rope, you try to manipulate events without snapping out of it completely.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.